sentry/src/transport.rs

```diff
@@ -222,9 +222,12 @@ implement_http_transport! {
                         }
                     }
 
+                    let mut body = vec![];
+                    envelope.to_writer(&mut body).unwrap();
```

Comment:
The `envelope.to_writer()` call also ends up calling some serde stuff that returns results instead of only writing into the vec (which is infallible).  Are you sure it's safe to unwrap those errors?

Reply:
(Yes, I am aware that differs from my comment on the test code of the envelope, but this isn't test code i gues)

Reply:
Actually there is so much unwrapping going on in the transports, I don’t think its worth caring about this I think.